### PR TITLE
Add PointingInfo class

### PR DESCRIPTION
**Pull request by cdeil**
This PR adds a `gammapy.data.PointingInfo` class.

I added a test file with a `POINTING` extension just now here:
https://github.com/gammapy/gammapy-extra/blob/master/test_datasets/hess_event_list.fits
The writing of the spec for this and debugging of the HESS pointing table is still work in progress.

My plan is to add some basic tests and [this check](https://github.com/cdeil/astrometric_checks/blob/master/test_pointing.py#L11) in this PR, but then merge and make an issue listing future work (e.g. interpolation of AZ / ALT to arbitrary times, format and content validation).

### CODE REVIEW

**Comment by cdeil** [comment]
😄  100% test coverage for new code. 😄

**Comment by cdeil** [comment]
@lmohrmann Do you have time for a quick review of this PR before I merge it?

**Comment by cdeil** [comment]
I've fixed up a few docstrings and added an example to the class docstring.
Have to move on to other things ... I'm merging this now.

File: gammapy/data/pointing.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    ----------
+    table : `~astropy.table.Table`
+        Table (with meta header info) on pointing
+
+    Examples
+    --------
+
+    >>> from gammapy.data import PointingInfo
+    >>> pointing_info = PointingInfo.read('$GAMMAPY_EXTRA/test_datasets/hess_event_list.fits')
+    >>> print(pointing_info)
```
Comment (JonathanDHarris) [inline gammapy/data/pointing.py]: I have a problem running this code

```
>>> from gammapy.data import PointingInfo
>>> pointing_info = PointingInfo.read('$GAMMAPY_EXTRA/test_datasets/hess_event_list.fits')
>>> print(pointing_info)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "gammapy/data/pointing.py", line 71, in __str__
    ss += 'Time ref:     {}\n'.format(self.time_ref.fits)
  File "/usr/local/lib/python2.7/dist-packages/astropy-1.0.4-py2.7-linux-x86_64.egg/astropy/time/core.py", line 725, in __getattr__
    return self.__getattribute__(attr)
AttributeError: 'Time' object has no attribute 'fits'
>>> pointing_info
<gammapy.data.pointing.PointingInfo object at 0x7f1c5e2a8990>
>>> pointing_info.time_ref
<Time object: scale='tt' format='mjd' value=51910.0007429>
>>> 
```

Comment (cdeil) [inline gammapy/data/pointing.py]: Your traceback shows that you're running with Astropy 1.0.4
The `fits` format for Astropy `Time` was added in Astropy 1.1.

Please update your Astropy to 1.1 or even better 1.2.

**Pull request merged**